components/style/properties/gecko.mako.rs

```diff
@@ -1964,6 +1965,21 @@ fn static_assert() {
         self.gecko.mTextEmphasisStyle = other.gecko.mTextEmphasisStyle;
     }
 
+    <%call expr="impl_color_setter('_webkit_text_fill_color', 'mWebkitTextFillColor')"></%call>
+    <%call expr="impl_color_copy('_webkit_text_fill_color', 'mWebkitTextFillColor')"></%call>
+    <%call expr="impl_color_clone('_webkit_text_fill_color', 'mWebkitTextFillColor')"></%call>
+
+    <%call expr="impl_color_setter('_webkit_text_stroke_color', 'mWebkitTextStrokeColor')"></%call>
+    <%call expr="impl_color_copy('_webkit_text_stroke_color', 'mWebkitTextStrokeColor')"></%call>
+    <%call expr="impl_color_clone('_webkit_text_stroke_color', 'mWebkitTextStrokeColor')"></%call>
```

Comment:
You don't need to implement gecko glues for predefined types. Helper does that for you. You can set gecko struct name like [this one](https://github.com/servo/servo/blob/61431b7fd61604da9a54b9dd6714438f6485bf7b/components/style/properties/longhand/column.mako.rs#L284) if it's not usual.